### PR TITLE
Minor cleanup for missing translations and removing unused components.

### PR DESCRIPTION
**Pull request by wtritch**
I just did a quick visual scan of our .html resources.  I added some missing translations and removed some unused widgets like the language selector.

### CODE REVIEW

File: webapp/src/main/webapp/src/app/app.component.html [hunk 1]
```diff
@@ -25,10 +25,6 @@
                     <i class="fa fa-user-circle"></i> <span *ngIf="user">{{'welcome' | translate:user }}</span> <span class="caret"></span>
                   </button>
                   <ul *dropdownMenu class="dropdown-menu" role="menu">
-                    <li role="menuitem"><a class="dropdown-item" href="#">Action</a></li>
-                    <li role="menuitem"><a class="dropdown-item" href="#">Another action</a></li>
-                    <li role="menuitem"><a class="dropdown-item" href="#">Something else here</a></li>
-                    <li role="separator" class="divider dropdown-divider"></li>
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/app.component.html]: We don't need placeholders in the user menu this close to UAT.

File: webapp/src/main/webapp/src/app/app.component.html [hunk 2]
```diff
@@ -47,17 +43,6 @@
...
 
-<div class="hidden">
-  <h1>{{ 'HELLO' | translate }}</h1>
-  <!-- todo: refactor to directive/component -->
-  <select #languageSelect (change)="translate.use(languageSelect.value)">
-    <option *ngFor="let language of translate.getLangs()" [value]="language" [selected]="language === translate.currentLang">
-      {{ language }}
-    </option>
-  </select>
-</div>
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/app.component.html]: It didn't look like we're going to allow language selection any time soon.

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 3]
```diff
@@ -1,7 +1,9 @@
 {
   "welcome": "Welcome {{firstName}} {{lastName}}",
   "application": {
-    "title": "Reporting System"
+    "title": "Reporting System",
+    "home": "Reporting System Dashboard",
+    "footer": "\u00A9 The Regents of the University of California &ndash; Smarter Balanced Assessment Consortium"
```
Comment (wtritch) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: I had to use the unicode for the copyright character.  `@copy;` doesn't get converted into the character.

File: webapp/src/main/webapp/src/app/app.component.html (hunk 1 above)
Comment (jtreuting) [inline webapp/src/main/webapp/src/app/app.component.html]: thanks, i had that on my list to remove

File: webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.ts [hunk 4]
```diff
@@ -242,6 +242,14 @@ export class AssessmentResultsComponent implements OnInit {
     exportRequest.assessmentItems = this.filteredAssessmentItems;
     exportRequest.pointColumns = this.pointColumns;
     exportRequest.showAsPercent = this.showValuesAsPercent;
+
+    this.angulartics2.eventTrack.next({
+      action: 'Export ItemsByPointsEarned',
+      properties: {
+        category: 'Export'
+      }
+    });
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.ts]: I added metrics capturing for CSV export actions.

File: webapp/src/main/webapp/src/app/app.component.html (hunk 2 above)
Comment (jtreuting) [inline webapp/src/main/webapp/src/app/app.component.html]: nope

**Pull request merged**